### PR TITLE
[KYUUBI #1068] event log can't upload to hdfs

### PR DESCRIPTION
**Pull request by zhang1002**
bug:
#1068 

This version, event log don't support upload file to hdfs

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by zhang1002** [comment]
cc @yaooqinn @ulysses-you @zhouyifan279

**Comment by timothy65535** [comment]
hi @zhang1002, following steps will helpful for new patch
```
git checkout master
git pull --rebase origin master
git checkout -b <new branch>
```

File: externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/events/EventLoggingServiceSuite.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ import org.apache.kyuubi.operation.{JDBCTestUtils, OperationHandle}
 
 class EventLoggingServiceSuite extends WithSparkSQLEngine with JDBCTestUtils {
 
-  private val logRoot = Utils.createTempDir()
+  private val logRoot = "file:" + Utils.createTempDir()
```
Comment (timothy65535) [inline externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/events/EventLoggingServiceSuite.scala:35]: `logRoot` from `Path` type to `String` type, so we don't need to call `toString` again.

```
  private val logRoot = "file:" + Utils.createTempDir()
  private val currentDate = Utils.getDateFromTimestamp(System.currentTimeMillis())


  override def withKyuubiConf: Map[String, String] = Map(
    KyuubiConf.ENGINE_EVENT_LOGGERS.key -> s"$JSON,$SPARK",
    KyuubiConf.ENGINE_EVENT_JSON_LOG_PATH.key -> logRoot.toString,
    "spark.eventLog.enabled" -> "true",
    "spark.eventLog.dir" -> logRoot.toString
  )
```

Comment (zhang1002) [reply externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/events/EventLoggingServiceSuite.scala:37]: it's good.
i don't find this problem.

**Review by yaooqinn — APPROVED** [review]

**Pull request closed without merging**